src/Plugin/migrate/source/YAML.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+namespace Drupal\migrate_default_content\Plugin\migrate\source;
+
+use Drupal\migrate\MigrateException;
+use Drupal\migrate\Plugin\migrate\source\SourcePluginBase;
+use Drupal\migrate\Plugin\MigrationInterface;
+
+/**
+ * Source for YAML.
+ *
+ * If the YAML file contains non-ASCII characters, make sure it includes a
+ * UTF BOM (Byte Order Marker) so they are interpreted correctly.
+ *
+ * @MigrateSource(
+ *   id = "yaml"
```

Comment:
There is only one ticket and is about the D8 port 

https://www.drupal.org/node/2808627
Pointing to a sandbox
http://cgit.drupalcode.org/sandbox-stborchert-2808617/tree/src/Plugin/migrate/source/Yaml.php

I feel that is wiser to wait a bit for the module maintainer to answer and open a branch. Meanwhile we can keep the source plugin here.

Can we make a copy or at least compatible with the one in the sandbox?